main.js

```diff
@@ -97,8 +87,36 @@ class Timer {
         return this.started;
     }
 }
-const imgs = [];
-for(let i=0;i<12;i++){
-    imgs.push(document.getElementById('img'+i));
+
+class DOM_Manager {
+    constructor() {
+        this.cardsCountVar = document.getElementsByClassName("card").length;
+        this.imgs = [];
+        for(let i=0;i<this.cardsCountVar;i++){
+            this.imgs.push(document.getElementById("img"+i));
+        }
+    }
+    setListener(fun) {

```

Comment:
The more common way to name the callback function is `cb` or `fn`. `fun` is fun :)